its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -1169,22 +1201,54 @@ void checkMultiLanguageSupportAngular() throws Exception {
+      tuple("csharpsquid:S4487", "MultiLanguageSupportAngular:Pages/Error.cshtml.cs")));
+    if (version.getMajor() == 8) {
+      expectedIssues.addAll(List.of(
+        tuple("csharpsquid:S3903", "MultiLanguageSupportAngular:Pages/Error.cshtml.cs"),
+        tuple("csharpsquid:S3903", "MultiLanguageSupportAngular:Controllers/WeatherForecastController.cs"),
+        tuple("csharpsquid:S3903", "MultiLanguageSupportAngular:WeatherForecast.cs")));
+    }
+    if (version.isGreaterThan(8, 9)) {
+      expectedIssues.addAll(List.of(
+        tuple("typescript:S1874", "MultiLanguageSupportAngular:ClientApp/src/app/fetch-data/fetch-data.component.ts"),
+        tuple("typescript:S125", "MultiLanguageSupportAngular:ClientApp/src/environments/environment.ts"),
+        tuple("typescript:S125", "MultiLanguageSupportAngular:ClientApp/src/polyfills.ts"),
+        tuple("typescript:S125", "MultiLanguageSupportAngular:ClientApp/src/polyfills.ts")));
+    }
+    if (version.isGreaterThanOrEquals(2025, 1)) {
+      expectedIssues.add(tuple("csharpsquid:S6966", "MultiLanguageSupportAngular:Program.cs"));
+    }
+
+    assertThat(issues)
+      .filteredOn(x -> !(x.getRule().startsWith("css") || x.getRule().startsWith("python")))
+      .extracting(Issue::getRule, Issue::getComponent)
+      .containsExactlyInAnyOrder(expectedIssues.toArray(new Tuple[]{}));
+
+    assertThat(issues)
+      .filteredOn(x -> x.getRule().startsWith("python"))
       .extracting(Issue::getRule, Issue::getComponent)
       .contains(
-        tuple("javascript:S3358", "MultiLanguageSupportAngular:ClientApp/proxy.conf.js"),
-        tuple("csharpsquid:S4487", "MultiLanguageSupportAngular:Controllers/WeatherForecastController.cs"),
-        tuple("csharpsquid:S4487", "MultiLanguageSupportAngular:Pages/Error.cshtml.cs"),
-        // tuple("csharpsquid:S6966", "MultiLanguageSupportAngular:Program.cs"), // Only reported on some versions of SQ.
-        // Some css, less, scss and python files are analyzed in node_modules. This is because the IT
-        // are running without scm support. Normally these files are excluded by the scm ignore settings.
-        // js/ts files in node_modules are additionally excluded by sonar.javascript.exclusions or sonar.typescript.exclusions
-        // and are therefore not reported here.
-        tuple("css:S4649", "MultiLanguageSupportAngular:ClientApp/node_modules/serve-index/public/style.css"),
-        tuple("css:S4654", "MultiLanguageSupportAngular:ClientApp/node_modules/less/test/browser/less/urls.less"),
-        tuple("css:S4654", "MultiLanguageSupportAngular:ClientApp/node_modules/bootstrap/scss/forms/_form-check.scss"),
-        tuple("python:S5754", "MultiLanguageSupportAngular:ClientApp/node_modules/flatted/python/flatted.py"));
+        tuple("python:S5754", "MultiLanguageSupportAngular:ClientApp/node_modules/flatted/python/flatted.py")
+      )
+      .size()
+      .isGreaterThan(1000); // 8.9 = 1053, 9.9 = 1210, 2025.1 = 1234
```

Comment:
Following my previous comment:
```suggestion
      .isIn(1053, 1212, 1234); // 8.9 = 1053, 9.9 = 1210, 2025.1 = 1234
```